pkg-manager/plugin-commands-installation/src/remove.ts

```diff
@@ -174,6 +174,7 @@ export async function handler (
   const store = await createOrConnectStoreController(opts)
   const removeOpts = Object.assign(opts, {
     ...getOptionsFromRootManifest(opts.rootProjectManifestDir, opts.rootProjectManifest ?? {}),
+    linkWorkspacePackagesDepth: opts.linkWorkspacePackages === 'deep' ? Infinity : opts.linkWorkspacePackages ? 0 : -1,
```

Comment:
Update since the draft was created: after poking at it a bit, I noticed that the single project codepath does not pass on `linkWorkspacePackagesDepth`, in the same way that the recursive codepath does. Making this change fixes the test, and the rest of the suite seems happy as well.

Sorry for the changing direction, I will stop pushing updates and wait a bit 😌